test/hotspot/jtreg/ProblemList.txt

```diff
@@ -132,7 +132,7 @@ runtime/AccModule/ConstModule.java 8294051 generic-all
 runtime/valhalla/inlinetypes/CircularityTest.java 8349037 generic-all
 runtime/valhalla/inlinetypes/verifier/StrictInstanceFieldsTest.java CODETOOLS-7904031 generic-all
 runtime/valhalla/inlinetypes/verifier/StrictStaticFieldsTest.java CODETOOLS-7904031 generic-all
-compiler/jvmci/jdk.vm.ci.runtime.test/src/jdk/vm/ci/runtime/test/TestResolvedJavaType.java 8357785 generic-all
+runtime/cds/appcds/RewriteBytecodesInlineTest.java 8361082 generic-all
```

Comment:
This issue is fixed with #1498 , so this seems like a merge conflict, right?

Reply:
Exactly.